src/components/select-project/select-project.controller.ts

```diff
@@ -73,7 +73,9 @@ export class SelectProjectController implements angular.IController {
       this.ctrl.existingProjectNames = _.map(this.ctrl.projects, 'metadata.name');
 
       // get most recently created
-      this.ctrl.selectedProject = this.$filter('mostRecent')(this.ctrl.projects);
+      if (!this.ctrl.selectedProject) {
```

Comment:
`this.ctrl.selectedProject` is bindable on the `<select-project>` component.  Initially it is undefined.  I've tested this in my local environment and the selected project is initially set to the most recently created project, and once another (or the same) selected-project is set, it remains 'selected' between wizard steps.